lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -47,8 +47,12 @@ $(document).ready(function() {
   $('#delete-link').click( function(e) {
     var ok = confirm($(this).data('confirm'));
     if ( ok ) {
-      var loc = '<%= delete_path %>/' + pageFullPath;
-      window.location = loc;
+      $.post('<%=delete_path %>/' + pageFullPath,
+        {},
+        function (result) {
+          // page successfully deleted, return to landing page
+          window.location = '/';
```

Comment:
The route itself is redirecting to `/pages`. I'm fine with either behavior really, but I'm not sure what will end up happening here, as the `delete` route is being triggered, but the ajax call is redirecting somewhere else?

Reply:
This works, but I agree that it isn't very pretty. Going forward, I think the `/pages` view should be refactored anyway so that the delete there also uses ajax (so that the overview is updated without reloading the entire page). Until then, I suggest we keep it like this.

Reply:
@dometto Is this okay with you?

Reply:
:+1: sounds right